src/python/CRABClient/CredentialInteractions.py

```diff
@@ -300,7 +300,14 @@ def createNewMyProxy(self, timeleftthreshold=0, nokey=False):
         myproxytimeleft = 0
         self.logger.debug("Getting myproxy life time left for %s" % self.defaultDelegation["myProxySvr"])
         # return an integer that indicates the number of seconds to the expiration of the proxy in myproxy
-        myproxytimeleft = myproxy.getMyProxyTimeLeft(serverRenewer=True, nokey=nokey)
+        # Also catch the exception in case WMCore encounters a problem with the proxy itself (one such case was #4532)
+        try:
+            myproxytimeleft = myproxy.getMyProxyTimeLeft(serverRenewer=True, nokey=nokey)
+        except Exception as ex:
+            logging.exception("Problems calculating proxy lifetime, logging stack trace and raising ProxyCreationException")
+            msg = ex._message if hasattr(ex, "_message") else str(ex)
```

Comment:
Seems it is coming from `WMException.py`. Maybe it does not hurt having this line after all. Let's just put a comment that it's there because WMException set it.